packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -202,7 +222,7 @@ class SystemRulesTable extends React.Component {
         } else {
             // One rule was selected
             const index = ((page - 1) * itemsPerPage * 2) + Number(key);
-            if (!currentRows[index].cells[4].title.props.unsupported) {
+            if (currentRows[index].cells[REMEDIATIONS_COLUMN].original) {
```

Comment:
This would not work since we removed the `<Ansible>` component, we no longer have an `unsupported` prop